Config.pde

```diff
@@ -0,0 +1,9 @@
+static class Config {
+  //static int width = 500;
+  //static int height = 690;
+  static String backendIp = "192.168.1.101";
```

Comment:
Oups yes, I forgot to remove my test ip

On Mon, Apr 20, 2015 at 12:30 AM, mgoimam <notifications@github.com> wrote:

> In Config.pde
> <https://github.com/firetetris/firetetris/pull/11#discussion_r28663576>:
>
> > @@ -0,0 +1,9 @@
> > +static class Config {
> > +  //static int width = 500;
> > +  //static int height = 690;
> > +  static String backendIp = "192.168.1.101";
>
> Default IP should be 127.0.0.1.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/firetetris/firetetris/pull/11/files#r28663576>.
>